renderdoc/driver/metal/metal_resources.h

```diff
@@ -127,6 +128,28 @@ struct UnwrapHelper
 METALCPP_WRAPPED_PROTOCOLS(WRAPPED_TYPE_HELPERS)
 #undef WRAPPED_TYPE_HELPERS
 
+namespace MetalResources
```

Comment:
What's the motivation for this namespace? The metal is implicit given the location of this code, and 'resources' is a bit generic and possibly misleading because the class in here (and presumably others that will follow) are _not_ resources themselves but rather resource-specific cold data.